installation.html

```diff
@@ -53,7 +53,7 @@
 
 <h1>Installing Gregorio</h1>
 
-<p>To install a complete Gregorio system, refer to the following pages:</p>
+<p>If you have a full installation of <span class="tex">T<span class="epsilon">e</span>X</span>Live 2016 (including MacTeX), then Gregorio 4.1 is already installed on your system as part of that installation.  If you use a different <span class="tex">T<span class="epsilon">e</span>X</span> distribution, have an older version of <span class="tex">T<span class="epsilon">e</span>X</span>Live, or want a newer version of Gregorio, then refer to the following pages:</p>
```

Comment:
I think it should also be mentionned that even if users don't have a complete TeXLive installation but only a partial one, they can complete it with Gregorio by installing the "music" bundle (I think)